src/stylesheets/screens/loader-details.scss

```diff
@@ -39,63 +10,70 @@
   }
 
   &__date .react-loading-skeleton {
-    border-radius: rem(12px);
-    height: rem(18px);
     width: rem(117px);
+    height: rem(18px);
+
+    border-radius: rem(12px);
   }
 
   &__day {
     margin-top: rem(14px);
   }
 
   &__day .react-loading-skeleton {
-    border-radius: 12px;
-    height: rem(18px);
     width: rem(90px);
+    height: rem(18px);
+
+    border-radius: 12px;
```

Comment:
@rafayet-monon 🙌   `rem()` here.

Reply:
Fixed in [6ada2d2](https://github.com/rafayet-monon/surveyor/pull/13/commits/6ada2d22d37cb0fa39af1ba8e59a8880825ee44e)